### DISCUSSION TITLE
Some new suggestions

### DISCUSSION THREAD

**Discussion by LoneStarz** [keiwando/toonsquid#35] [Ideas]
I’ve made a list before, I’m just adding these two to the requests I already made!

the ability to copy your current selection.
An option to anchor things together in both drawing layers and the timeline. In some occasions it would be useful when you have multiple drawings but you want it to have the same movement to another part in a different layer

**Comment by LoneStarz** [comment]
I didn’t mention it before in my other post because I just came up with this but the mesh transform tool would be useful in pixel layers if you could edit it like a keyframe, I don’t know if it’s possible but that would have so much potential

**Reply by trinhpxl** [reply]
yeah ik that copy pasting reigon is coming in the next update (1.7) and mesh is planned, too
idk abt the rest tho LOL
while at it, is it possible to have motion blur on toonsquid ? 🦡

**Reply by keiwando** [reply]
I'm planning to add a regular gaussian blur at some point. Automatic motion blur is more tricky but I'll gladly look into it eventually.

**Reply by LoneStarz** [reply]
Gaussian blur would be a welcoming addition too!, sometimes I use it to make my lighting softer on procreate so it would be really cool if toonsquid also had this feature!

**Comment by keiwando** [comment]
> An option to anchor things together in both drawing layers and the timeline.

Do you mean temporarily so that you can select them all and for example move multiple pixel layers at once or permanently so that you can animate them together with keyframes?

**Reply by LoneStarz** [reply]
Yes! Is there a way to do that already?

**Reply by keiwando** [reply]
The former, no, but the latter, yes, using the transform hierarchy: https://toonsquid.com/handbook/transform/hierarchy/

**Reply by LoneStarz** [reply]
Oh thank you! I haven’t looked into this feature so I’m going to try it later!

**Comment by rdc457** [comment]
Can u guys PLEASE add merge when it comes to making animations, or select all and do keyframing with them

**Reply by keiwando** [reply]
If you are just looking to animate the position, scale and rotation of drawings in multiple animation layers at once, you can already use the transform hierarchy for that.
https://toonsquid.com/handbook/transform/hierarchy/

**Reply by keiwando** [reply]
The demo project uses that for animating the combined movement of all of the birds for example.

**Reply by rdc457** [reply]
Oh ok

**Comment by rdc457** [comment]
Add skew

**Comment by rdc457** [comment]
Can u guys make a tutorial video about transform hierarchy
